prefactor.cwl

```diff
@@ -15,62 +15,86 @@ inputs:
 outputs:
   losoto_h5:
     type: File
+    format: h5parm
```

Comment:
Is this file format documented anywhere? Even a sparse GitHub wiki page would be better than `h5parm`